modules/util/portal-tools-sample-sql-builder/src/main/resources/com/liferay/portal/tools/sample/sql/builder/dependencies/asset_publisher.ftl

```diff
@@ -18,10 +18,16 @@
 <#list pageCounts as pageCount>
 	<#assign
 		portletId = dataFactory.getPortletId("com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_")
-
+		assetListEntryModel = dataFactory.newAssetListEntryModel(groupId, pageCount)
```

Comment:
@slnn Should we sort these?

Reply:
How about this? Sort and group these models alphabetically

```
<#assign
		portletId = dataFactory.getPortletId("com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_")

		assetListEntryModel = dataFactory.newAssetListEntryModel(groupId, pageCount)
		layoutModel = dataFactory.newLayoutModel(groupId, groupId + "_asset_publisher_" + pageCount, "", portletId)
	/>
```

Reply:
Update to the second commit` LPD-45665 Generate AssetListEntry for each AssetPublisher portlet
`